eosdatadistribution/MDNS/__init__.py

```diff
@@ -39,7 +39,7 @@ def __init__(self):
         self.pretty_name = avahi.ServiceTypeDatabase.ServiceTypeDatabase()
 
     def get_human_type(self, type):
-        if self.pretty_name.has_key(type):
+        if type in self.pretty_name:
```

Comment:
Please split this out then -- this isn't part of a Python 2 to Python 3 port, 2to3 is just doing a buggy find/replace.